apps/poker-roll/src/main.tsx

```diff
@@ -1,13 +1,41 @@
 import { StrictMode } from 'react';
 import * as ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
+
+// import Cashgame 
+// import Bankroll 
+// import Tournament 
 
 import App from './app/app';
 
-const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
-);
-root.render(
-  <StrictMode>
-    <App />
-  </StrictMode>
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Root />,
```

Comment:
Looks like this component is missing. Planning on making this your layout component?

Reply:
changed to home and created home page